grafana/provisioning/dashboards/metrics.json

```diff
@@ -907,56 +915,571 @@
+                                },
+                                "insertNulls": false,
+                                "lineInterpolation": "linear",
+                                "lineWidth": 1,
+                                "pointSize": 5,
+                                "scaleDistribution": {
+                                    "type": "linear"
+                                },
+                                "showPoints": "auto",
+                                "spanNulls": false,
+                                "stacking": {
+                                    "group": "A",
+                                    "mode": "none"
+                                },
+                                "thresholdsStyle": {
+                                    "mode": "dashed"
+                                }
+                            },
+                            "mappings": [],
+                            "thresholds": {
+                                "mode": "absolute",
+                                "steps": [
+                                    {
+                                        "color": "green",
+                                        "value": null
+                                    }
+                                ]
+                            },
+                            "unit": "none"
+                        },
+                        "overrides": []
+                    },
+                    "gridPos": {
+                        "h": 8,
+                        "w": 24,
+                        "x": 0,
+                        "y": 36
+                    },
+                    "id": 23,
+                    "options": {
+                        "legend": {
+                            "calcs": [],
+                            "displayMode": "table",
+                            "placement": "right",
+                            "showLegend": true
+                        },
+                        "tooltip": {
+                            "mode": "single",
+                            "sort": "none"
+                        }
+                    },
+                    "targets": [
+                        {
+                            "datasource": {
+                                "type": "prometheus",
+                                "uid": "${ds}"
+                            },
+                            "editorMode": "code",
+                            "expr": "sum(node_filefd_allocated{instance=~\"$node\"}) by(instance)",
+                            "instant": false,
+                            "legendFormat": "{{instance}}",
+                            "range": true,
+                            "refId": "A"
+                        }
+                    ],
+                    "title": "Number of allocated file descriptors by node ($node)",
+                    "type": "timeseries"
+                }
+            ],
+            "title": "Node stats",
```

Comment:
You could think about the presentation of those graphs. Rn they are one under another and it's giving repeatable and a bit boring (IMHO).

Reply:
Yeah, it is boring, but I don't know if Grafana dashboards must be entertaining. When I need entertainment, I read books, play games, and definitely not read Grafana dashboards 😆 .

Reply:
Lol, you do what? Don't tell me you don't look at Grafana dashboards when bored :scream: